thenewboston_node/accounts/views/transactions.py

```diff
@@ -0,0 +1,53 @@
+from drf_spectacular.utils import OpenApiParameter, extend_schema
+from more_itertools import ilen
+from rest_framework.exceptions import ParseError
+from rest_framework.mixins import ListModelMixin
+from rest_framework.viewsets import GenericViewSet
+
+from thenewboston_node.accounts.serializers.transaction import TransactionSerializer
+from thenewboston_node.business_logic.blockchain.base import BlockchainBase
+from thenewboston_node.business_logic.models import CoinTransferSignedChangeRequest
+from thenewboston_node.core.pagination import CustomLimitOffsetPagination
+from thenewboston_node.core.utils.itertools import SliceableCountableIterable
+
+
+class TransactionViewSet(ListModelMixin, GenericViewSet):
+    serializer_class = TransactionSerializer
+    pagination_class = CustomLimitOffsetPagination
+
+    @extend_schema(
+        parameters=[
+            OpenApiParameter('ordering', location=OpenApiParameter.QUERY, description='asc/desc', default='asc'),
+        ]
+    )
+    def list(self, request, id):  # noqa: A002, A003
+        return super().list(request, id)
+
+    def get_queryset(self):
+        account_id = self.kwargs.get('id')
+        ordering = self.request.query_params.get('ordering', 'asc')
+
+        if ordering == 'asc':
```

Comment:
Use ordering that mimics DRF standard way. See implementation here: https://github.com/thenewboston-developers/thenewboston-node/pull/364